scrapper.py

```diff
@@ -1,10 +1,140 @@
 import requests
 from bs4 import BeautifulSoup
+from sys import argv
+import re
 
-page = requests.get('http://127.0.0.1:5500/krisha_scrapper/flat_list.html')
-soup = BeautifulSoup(page.content, 'html.parser')
+def check_cli_arg(command_line_arguments: list) -> str:
```

Comment:
@ProvVlad, take a look at this coding principle which is called "Don't repeat yourself" [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). I would rewrite this function in the following way:

```suggestion
def terminate_script(exit_code: int)
    OUTPUT_MESSAGES = {1:"You missed an argument. It's expected a URL that should be grabbed.",
        2:"Argument have to begin from 'https://krisha.kz'.",
        3:"Too many arguments. It's expected just one argument which is URL to be grabbed."}
    
    print(OUTPUT_MESSAGES[exit_code])
    exit(1)

def check_cli_arg(command_line_arguments: list) -> str:
    
    if len(command_line_arguments) == 1:
        terminate_script(1)
    elif len(command_line_arguments) == 2:
        cli_argument = command_line_arguments[1]
        if not re.search(re.compile('^https://krisha.kz'), cli_argument.strip()):
            terminate_script(2)
    elif len(command_line_arguments) > 2:
        terminate_script(3)

    return cli_argument.strip()
```